00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
Where else would the "share" folder be? I could not find any in tools/rest_api?
So that change was required to make the rest_api actually work for us (with autoproj v2) since otherwise the plugins are not found.

Why should an env var not refer to the build dir? Sounds to me a bit arbitrary.